keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -887,6 +916,16 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+Same as above: all the components must have support for it to work.
+Specifically, kube-apiserver will filter this field if it doesn't recognize/support it.
+If the kubelet doesn't support the field, it will not pull the image, and the path will not be present for the volume manager
+to mount it in, and the pod will fail.
```

Comment:
for oci volumes .. does it apply that the volume manager will mount?.. in this case isn't the container runtime the volume manager..

Reply:
> for oci volumes .. does it apply that the volume manager will mount?

please see https://github.com/kubernetes/enhancements/pull/4897#discussion_r1794485855